HPHBhelper/hogwarts/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class Games(models.Model):
+    shortcode = models.CharField(max_length=4)
+    isActive = models.BooleanField(default=False)
+    host = models.ForeignKey(Players)
+
+    class TurnOrder(models.Model):
+        #The turn order is set after a game is created but before it starts
+        pass
+
+
+class Players(models.Model):
+    name = models.CharField(max_length=20)
+    game = models.ForeignKey(Games)
+
+    class Characters(models.IntegerChoices):
+        HARRY_POTTER = 1
+        RON_WEASLEY = 2
+        HERMIONE_GRANGER = 3
+        GINNY_WEASLEY = 4
+        LUNA_LOVEGOOD = 5
+
+    player_character = models.IntegerField(choices=Characters.choices)
```

Comment:
I would expect a `ChoiceField` here so no need for the class here, just a list or tuple with choice tuples, e.g. https://www.geeksforgeeks.org/choicefield-django-forms/